Improve dbt-trino connection overview page (#3222)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/connect-data-platform/connect-starburst-trino.md

---
title: "Connect Starburst or Trino"
description: "Configure Starburst or Trino connection."
sidebar_label: "Connect Starburst or Trino"
---

The following are the required fields for setting up a connection with a [Starburst Enterprise](https://docs.starburst.io/starburst-enterprise/index.html), [Starburst Galaxy](https://docs.starburst.io/starburst-galaxy/index.html), or [Trino](https://trino.io/) cluster. Unless specified, "cluster" means any of these products' clusters.

| Field | Description | Examples |
| --- | --- | --- |
| **Host** | The hostname of your cluster. Don't include the HTTP protocol prefix. | `mycluster.mydomain.com` |
| **Port** | The port to connect to your cluster. By default, it's 443 for TLS enabled clusters. | `443` |

> | **User** | The username (of the account) to log in to your cluster. When connecting to Starburst Galaxy clusters, you must include the role of the user as a suffix to the username.<br/><br/> | Format for Starburst Enterprise or Trino depends on configured authentication type. <br/>Format for Starburst Galaxy:<br/> <ul><li>`user.name@mydomain.com/role`</li></ul> |

Reviewer: Cleaned up the table a tiny bit. 

```suggestion
| **User** | The username (of the account) to log in to your cluster. When connecting to Starburst Galaxy clusters, you must include the role of the user as a suffix to the username.<br/><br/> | Format for Starburst Enterprise or Trino depends on your configured authentication method. <br/><br/>Format for Starburst Galaxy:<br/> <ul><li>`user.name@mydomain.com/role`</li></ul> |
```

Author: Fixed now .. just fyi .. in the Trino context we support multiple "authentication types" . but method works just fine in this context as well

sidebar_label: "Connect Starburst/Trino"
---

> The following are the required fields for setting up a connection with a [Starburst Enterprise](https://docs.starburst.io/starburst-enterprise/index.html), [Starburst Galaxy](https://docs.starburst.io/starburst-galaxy/index.html), or [Trino](https://trino.io/) cluster. Unless specified, "cluster" means any of these products' clusters.

Reviewer: going off of past experience, future tense is ok in this situation since it is referring to what the person is going to encounter. 

```suggestion
The following are the required fields for setting up a connection with a [Starburst Enterprise](https://docs.starburst.io/starburst-enterprise/index.html), [Starburst Galaxy](https://docs.starburst.io/starburst-galaxy/index.html), and [Trino](https://trino.io/) cluster. Unless specified, "cluster" will mean any of these products' clusters.
```

Author: That typically goes against style guides and I would prefer if we can avoid future tense. Also specifically in this context the future tense is wrong ... the meaning is in the docs right now.

Reviewer: I've supported several style guides and have found from experience that it depends. Some of those experiences included specialized technical editors, too. totally a non-blocker for me! 🚀

| Field | Description | Examples |
| --- | --- | --- |
| **Host** | The hostname of your cluster. Don't include the HTTP protocol prefix. | `mycluster.mydomain.com` |
| **Port** | The port to connect to your cluster. By default, it's 443 for TLS enabled clusters. | `443` |
| **User** | The username (of the account) to log in to your cluster. When connecting to Starburst Galaxy clusters, you must include the role of the user as a suffix to the username.<br/><br/> | Format for Starburst Enterprise or Trino depends on configured authentication type. <br/>Format for Starburst Galaxy:<br/> <ul><li>`user.name@mydomain.com/role`</li></ul> |
| **Password** | The user's password. |   |

> | **Database** | The name of a catalog in your cluster. | `my_postgres_catalog` |

Author: Is this optional? If yes, we should add that info. Also we should not use possessive example values .. how about just `example_catalog`

Reviewer: @mosabua : just double-checked this. it's required.

> | **Database** | The name of a catalog in your cluster. | `my_postgres_catalog` |

Reviewer 2: ```suggestion
| **Database** | The name of a catalog in your cluster. | `example_postgres_catalog` |
```

Author: Why do you want to keep postgres in it .. the technology used by the data source is less relevant than the actual data.

> | **Schema** | The name of a schema in your cluster that exists within the specified catalog.  | `my_schema` |

Author: Is this optional? If yes, we should add that info. Also we should not use possessive example values .. how about just `example_schema`

Reviewer 2: ```suggestion
| **Schema** | The name of a schema in your cluster that exists within the specified catalog.  | `example_schema` |
```

Reviewer: this is also required. thanks for checking, @mosabua !
